addons/addon-raas-s3-copy/packages/s3-synchronizer/README.md

```diff
@@ -24,8 +32,12 @@ Usage of bin/s3-synchronizer-darwin-amd64:
         Whether to print debug information
   -destination string
         The directory to download to (default "./")
-  -pollInterval int
-        The delay (in seconds) between api http requests (default 60)
+  -recurringDownloads 
+        Whether to periodically download changes from S3 (default false)
+  -stopRecurringDownloadsAfter int
+        Stop recurring downloads after certain number of seconds. ZERO or Negative value means continue indefinitely. (default -1 i.e., indefinitely)
+  -downloadInterval int
+        The interval at which to re-download changes from S3 in seconds. This is only applicable when recurringDownloads is true. (default 60)
```

Comment:
nit: Probably include that download/sync time from S3 is not counted towards wait time, and regardless of how much time the re-sync took we will wait for these many seconds.

Reply:
Done. Added the following line

> Note that this does not include the download time. This specifies the duration in seconds to wait before initiating the next download after the previous one completes.